<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="Rluer" name="rluer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelection"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionText"/>
   <addaction name="actionLine"/>
   <addaction name="actionText2"/>
  </widget>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/image/pointer.png</normaloff>:/image/pointer.png</iconset>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="toolTip">
    <string>鼠标</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/image/circle.png</normaloff>:/image/circle.png</iconset>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="toolTip">
    <string>圆形</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/image/rectangle.png</normaloff>:/image/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="toolTip">
    <string>矩形</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/image/textpointer.png</normaloff>:/image/textpointer.png</iconset>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="toolTip">
    <string>常规文本</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/image/linepointer.png</normaloff>:/image/linepointer.png</iconset>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="toolTip">
    <string>线段</string>
   </property>
  </action>
  <action name="actionText2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/image/textpointer.png</normaloff>:/image/textpointer.png</iconset>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="toolTip">
    <string>变形文本</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Rluer</class>
   <extends>QWidget</extends>
   <header location="global">UWidget\rluer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
